docs/0.3-DRAFT/index.md

```diff
@@ -1378,7 +1328,90 @@ To record curation actions which modify a [File](http://schema.org/MediaObject)
 }
 ```
 
+### Workflows and scripts
+
+Scientific workflows and scripts that can be or were used to analyze or generate files contained in an the RO-Crate MAY be embedded in an RO-Crate. Workflows and scripts are described using the type SoftwareSourceCode instead of `SoftwareApplication` above.  
+
+The distinction is fluid, and comes down to the availability and understandability of code to re-run a workflow. For instance, office spreadsheet applications are generally available and do not need further explanation; while a Python script that is customized for a particular data analysis might be important to understand further and SHOULD be included as source code in the RO-Crate dataset:
+
+```json
+       {
+            "@id": "workflow/retropath.knime",  
+            "@type": ["File", "SoftwareSourceCode", "Workflow"],
+            "name": "RetroPath Knime workflow",
+            "description": "KNIME implementation of RetroPath2.0 workflow",
+            "author": {"@id": "https://orcid.org/0000-0001-2345-6789"},
+            "programmingLanguage": {"@id": "https://www.knime.com/knime-software/knime-analytics-platform"},
+            "license": "https://spdx.org/licenses/BSD-2-Clause.html",
+            "potentialAction": {
+                "@type": "ActivateAction",
+                "instrument": {"@id": "https://www.knime.com/knime-software/knime-analytics-platform"}
+            }
+        }
+```
+
+Workflows and scripts are saved on disk using a language and generally need a runtime, in RO-Crate this SHOULD be indicated using a liberal interpretation of `programmingLanguage` and `potentialAction`. Note that the language and its runtime might differ, e.g. there are multiple potential runtimes of a JavaScript, although in the base-case they are the same, and as a shortcut can be described in one go as both a `ComputerLanguage` and `SoftwareApplication`:
```

Comment:
This text mentioning `potentialAction` and `alternateName` below seems to reflect the Workflow section **before** my merged `tidy-workflow` from #49  - now I have to check everything else in all the moved sections to see if they are also outdated